app/ui/src/Pages/Projects/components/AdminEmail/AdminEmail.tsx

```diff
@@ -0,0 +1,26 @@
+import styles from './AdminEmail.module.scss';
+import CopyToClipboard from '../../../../Components/CopyToClipboard/CopyToClipboard';
+import { Button } from 'kwc';
+import IconSend from '@material-ui/icons/Send';
+import React from 'react';
+
+type Props = {
+  email: string;
+};
+function AdminEmail({ email }: Props) {
+  function onSendEmail() {
+    window.open(`mailto:${email}`);
+  }
+
+  return (
+    <div className={styles.adminEmail}>
+      <span>{email}</span>
+      <div className={styles.actions}>
+        <CopyToClipboard>{email}</CopyToClipboard>
+        <Button Icon={IconSend} label={''} onClick={() => onSendEmail()} />
```

Comment:
we can omit the arrow function in the onClick prop, like this:
```suggestion
        <Button Icon={IconSend} label={''} onClick={onSendEmail} />
```

Reply:
Done